site/public/css/server.css

```diff
@@ -1370,6 +1370,31 @@ end of styles used in the admin gradeable page
     white-space: nowrap;
 }
 
+/* prevents line break in the grade viewed check marks */
+.td-viewed-grade {
+    white-space: nowrap;
+}
+
+.viewed-green-check {
```

Comment:
Maybe some margin on the right and left but most left would look nice as the checks are hugging the left border.
![hugging_checks](https://user-images.githubusercontent.com/15002610/61389055-3e8ca700-a886-11e9-8674-1cfacdf5bc00.PNG)

Reply:
Added some padding to the right, I messed around with it on the left as well and don't think it is necessary.  Does it look better now? I can update it more if there are any other suggestions.